app/main/routes.py

```diff
@@ -52,6 +52,15 @@ def start_page():
     return redirect(url_for(".download"))
 
 
+@bp.route("/request-received", methods=["GET", "POST"])
+@login_required(return_app=SupportedApp.POST_AWARD_FRONTEND)
+def download_confirmation():
+    if not g.is_authenticated:
+        return render_template("login.html")
```

Comment:
I don't think we should be able to get this - the `login_required` decorator should protect us from this 👀